compose.yaml

```diff
@@ -50,6 +52,12 @@ services:
 volumes:
   nextcloud_aio_mastercontainer:
     name: nextcloud_aio_mastercontainer # This line is not allowed to be changed as otherwise the built-in backup solution will not work
+    # # Optional: Use a volume mount instead of internal docker volume
+    # driver: local
+    # driver_opts:
+    #   type: none
+    #   o: bind
+    #   device: ./nextcloud_data
```

Comment:
I am not sure if we should expose this. People might think that all data will be stored in that location which is not the case. I would rather point at https://github.com/nextcloud/all-in-one#how-to-store-the-filesinstallation-on-a-separate-drive

Reply:
I think those are two different topics. I personally just dislike docker volumes (because I tend to prune my whole docker system from time to time and don't want to risk loosing my data) and would prefer to mount all persistent data of the nextcloud installation in a local folder so that I can always access those files using the standard file system, just in case. This is not about changing the storage directory of docker. The datafolder and the compose file should be everything required to reproduce the current installation on a new server. Isn't this the case for the AIO installation?

Reply:
> The datafolder and the compose file should be everything required to reproduce the current installation on a new server. Isn't this the case for the AIO installation?

It isnt. See https://github.com/nextcloud/all-in-one#how-does-it-work. Also see https://github.com/nextcloud/all-in-one#backup-solution